src/mutable_arithmetics.jl

```diff
@@ -0,0 +1,92 @@
+#  Copyright 2017, Iain Dunning, Joey Huchette, Miles Lubin, and contributors
+#  This Source Code Form is subject to the terms of the Mozilla Public
+#  License, v. 2.0. If a copy of the MPL was not distributed with this
+#  file, You can obtain one at http://mozilla.org/MPL/2.0/.
+#############################################################################
+# JuMP
+# An algebraic modeling language for Julia
+# See http://github.com/JuliaOpt/JuMP.jl
+#############################################################################
+# src/mutable_arithmetics.jl
+# Implements the mutable arithmetics api defined in MutableArithmetics.jl for
+# `GenericAffExpr` and `GenericQuadExpr`.
+#############################################################################
+
+const GenericAffOrQuadExpr{C, V} = Union{GenericAffExpr{C, V}, GenericQuadExpr{C, V}}
+
+MA.mutability(::Type{<:GenericAffOrQuadExpr}) = MA.IsMutable()
+function MA.mutable_copy(expr::GenericAffOrQuadExpr)
```

Comment:
`mutable_copy` doesn't appear in the MA docs

Reply:
Now it does :)